src/cbmc/cbmc_parse_options.cpp

```diff
@@ -940,11 +940,15 @@ void cbmc_parse_optionst::help()
     "                              NULL in a recursively nondet initialized struct\n" /* NOLINT(*) */ \
     " --pointers-to-treat-as-array <identifier,...>  Comma separated list of\n" \
     "                               identifiers that should be initialized as arrays\n" /* NOLINT(*) */ \
+    " --pointers-to-treat-as-string <identifier,...> Comma separated list of\n" \
+    "                               identifiers that should be initialized as 0\n" \
+    "                               terminated strings\n" \
```

Comment:
Probably also want an option to enable/disable 'only printable-chars' as well?

Reply:
Maybe? It's ultimately just a matter of how many cases we want to support (only alphanumeric, only lower/uppercase/only matching this regex etc)

Reply:
Yeah, I think just a switch like that would be fine - basically gives a simple choice of "printable vs anything that fits in a char" (or rather "anything except null that fits in a char...)